Client/Frontend/Home/HomepageViewController.swift

```diff
@@ -163,17 +158,6 @@ class HomepageViewController: UIViewController, HomePanel, FeatureFlaggable, The
         }
     }
 
-    override func viewDidLayoutSubviews() {
-        super.viewDidLayoutSubviews()
-
-        // make sure the keyboard is dismissed when wallpaper onboarding is shown
-        // Can be removed once underlying problem is solved (FXIOS-4904)
```

Comment:
Seems like we can close [FXIOS-4904](https://mozilla-hub.atlassian.net/browse/FXIOS-4904) with this change?

Reply:
Closed